runtime/lua/vim/filetype.lua

```diff
@@ -774,6 +774,12 @@ local extension = {
   xml = function() vim.fn["dist#ft#FTxml"]() end,
   y = function() vim.fn["dist#ft#FTy"]() end,
   zsql = function() vim.fn["dist#ft#SQL"]() end,
+  txt = function(_, bufnr)
```

Comment:
Even though it's not used in this function, keep the `path` argument so that it's clear what the argument is on inspection:

```suggestion
  txt = function(path, bufnr)
```

Other people are going to copy this function for their contributions, so it is helpful to make it clear what the function arguments are.

Reply:
Buuuut the linter isn't happy about that.

Reply:
Yes, for now. See https://github.com/neovim/neovim/pull/16927

Reply:
Ok, [I see](https://github.com/neovim/neovim/pull/16927) :)

Reply:
That PR is now merged, if you wanted to wait and rebase on that.